java/debugger.jpda.truffle/src/org/netbeans/modules/debugger/jpda/truffle/actions/ToggleBreakpointsInLanguagesActionProvider.java

```diff
@@ -47,7 +47,7 @@ public class ToggleBreakpointsInLanguagesActionProvider extends ActionsProviderS
     
     private static final Set<String> IGNORED_MIME_TYPES = new HashSet<>(
             // We have JSLineBreakpoint in JavaScript
```

Comment:
This is telling Truffle not to handle Groovy, otherwise both Truffle and Groovy will be registered on the toggle breakpoint action. Due to a "bug" in the Debugger API Truffle wins which then do not place a breakpoint into Groovy.